src/transfers-history/adapters/db/transfers-repository.ts

```diff
@@ -64,7 +64,7 @@ export class TransfersRepository {
       ...transfer,
       filled,
       status: transfer.amount.eq(filled) ? "filled" : "pending",
-      fillTxs: [...transfer.fillTxs, fillTxHash],
+      fillTxs: Array.from(new Set([...transfer.fillTxs, fillTxHash])),
     };
```

Comment:
In some cases (request All transactions followed by requesting only My transactions) `fillTxs` were considered multiple times. Keep only the distinct values